crypto/client_test.go

```diff
@@ -0,0 +1,63 @@
+package crypto
+
+import (
+	"testing"
+)
+
+func TestNew(t *testing.T) {
+	fixtures := []struct {
+		t  string
+		fn func(interface{}) bool
+	}{
+		{
+			t: "",
+			fn: func(c interface{}) bool {
```

Comment:
Hmm, why did you decide to accept an empty interface here instead of a `Client` type? Applies to lines below too.

Reply:
No [good] reason. Updated, thanks!